src/utils/date.ts

```diff
@@ -0,0 +1,7 @@
+export const getLastYearDate = (date: Date) => {
```

Comment:
p3;

쉽게 유추할 순 있겠지만 return 타입을 명시해주게 좋을 것 같아요

```suggestion
export const getLastYearDate = (date: Date): Date => {
```